src/cookbook/check-status.md

```diff
@@ -14,6 +14,10 @@ head:
 ```rust
     // To get the status, the Client struct already has a function get_status(),
     // so you may just execute it.
+    // The result message will include a deserialized json object with parameters like:
+    //  Quantity of peers
+    //  Quantity of blocks
+    //  Quantity of accepted and rejected transactions and some more
     let status = iroha_client.get_status().unwrap();
     println!("{:?}", status);
```

Comment:
> The result message will include a deserialized json object with parameters like:...

Not sure about this being as expressive as just a snippet with an actual sample output:

```rs
println!("{status:?}");
// Sample output:
//
// Status {
//     peers: 4,
//     blocks: 5,
//     txs_accepted: 31,
//     txs_rejected: 3,
//     uptime: Uptime(
//         5.937s,
//     ),
//     view_changes: 2,
//     queue_size: 18,
// }

```